spec/Authentication/AuthenticationBehavior.php

```diff
@@ -1,11 +0,0 @@
-<?php
-
-namespace spec\Http\Message\Authentication;
-
-trait AuthenticationBehavior
```

Comment:
It did (partly). However PHPSpec currently does not support more complex examples (eg. it cannot use prophecy as function arguments) because of some sort of autoloading issues. So I decided to remove this because I though it didn't make sense just to avoid duplication of one example.